@uportal/notification-icon/src/components/NotificationItem.vue

```diff
@@ -48,7 +60,7 @@ export default {
             return () => {};
         },
         target() {
-            return this.redirectAction ? '_self' : '_blank';
+            return this.redirectAction || this.strategy === 'list' ? '_self' : '_blank';
```

Comment:
I believe we need to add `rel="noopener noreferrer"` everywhere we use `target="_blank"`, like here:  https://github.com/Jasig/uPortal/blob/master/uPortal-webapp/src/main/resources/layout/theme/respondr/navigation.xsl#L331